{"uid":"c843e2b48da4a994","name":"Create a new Order for new Customer","fullName":"TestOrder.TestOrder#test_create_new_user_order_4___frugal_latido_com_np____Test_123____9881012074____abced_Test_4____testuser174_mail_com____2023_05_29____Siliguri____female____Exemplar__male_____Bronze_Mist_Antique____42____Golden_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Skin_Tight____Test_002____2023_05_29___100900_","historyId":"3748e30bf56aba1f9f605e166aa9febe","time":{"start":1685023379935,"stop":1685023683333,"duration":303398},"status":"failed","statusMessage":"AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.","statusTrace":"self = <TestsCases.TestOrder.TestOrder testMethod=test_create_new_user_order_4___frugal_latido_com_np____Test_123____98810120...n_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Skin_Tight____Test_002____2023_05_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9881012074', name = 'abced Test 4', email = 'testuser174@mail.com'\ndob = '2023-05-29', address = 'Siliguri', gender = 'female'\nproductname = 'Exemplar (male)', leatherProfile = 'Bronze Mist Antique'\nleatherSize = '42', hardware = 'Golden Antique', lining = 'Black'\npolyfill = 'Yes', size = '40', armhole = 5.6, height = 4.4, shoulder = 5\nweight = 46, length = 8, arms = 12, hips = 11, chest = 7, waist = 1, sleeves = 9\nbodytype = 'Skin Tight', remark = 'Test 002', date = '2023-05-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for new Customer\")\n    @allure.story(\"Create an Order for new Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forNewCustomer\"))\n    @unpack\n    def test_create_new_user_order(self, username, password, contactNo, name, email, dob, address, gender, productname,\n                                   leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(self.driver, username, password)\n        try:\n            self.assertEqual(\n                self.Order.create_new_user_order(contactNo, name, email, dob, address, gender, productname,\n                                                 leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height,\n                                                 shoulder, weight, length, arms, hips, chest, waist, sleeves, bodytype, remark,\n                                                 date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Creating new Order for new customer.\",\n                          attachment_type=AttachmentType.PNG)\n            self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\n>           raise e\n\nTestOrder.py:80: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTestOrder.py:69: in test_create_new_user_order\n    self.assertEqual(\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestOrder","time":{"start":1685021911346,"stop":1685021911346,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.","statusTrace":"self = <TestsCases.TestOrder.TestOrder testMethod=test_create_new_user_order_4___frugal_latido_com_np____Test_123____98810120...n_Antique____Black____Yes____40___5_6__4_4__5__46__8__12__11__7__1__9___Skin_Tight____Test_002____2023_05_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9881012074', name = 'abced Test 4', email = 'testuser174@mail.com'\ndob = '2023-05-29', address = 'Siliguri', gender = 'female'\nproductname = 'Exemplar (male)', leatherProfile = 'Bronze Mist Antique'\nleatherSize = '42', hardware = 'Golden Antique', lining = 'Black'\npolyfill = 'Yes', size = '40', armhole = 5.6, height = 4.4, shoulder = 5\nweight = 46, length = 8, arms = 12, hips = 11, chest = 7, waist = 1, sleeves = 9\nbodytype = 'Skin Tight', remark = 'Test 002', date = '2023-05-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for new Customer\")\n    @allure.story(\"Create an Order for new Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forNewCustomer\"))\n    @unpack\n    def test_create_new_user_order(self, username, password, contactNo, name, email, dob, address, gender, productname,\n                                   leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(self.driver, username, password)\n        try:\n            self.assertEqual(\n                self.Order.create_new_user_order(contactNo, name, email, dob, address, gender, productname,\n                                                 leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height,\n                                                 shoulder, weight, length, arms, hips, chest, waist, sleeves, bodytype, remark,\n                                                 date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Creating new Order for new customer.\",\n                          attachment_type=AttachmentType.PNG)\n            self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\n>           raise e\n\nTestOrder.py:80: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTestOrder.py:69: in test_create_new_user_order\n    self.assertEqual(\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.","steps":[],"attachments":[{"uid":"79d83cf999a783fc","name":"Creating new Order for new customer.","source":"79d83cf999a783fc.png","type":"image/png","size":61531},{"uid":"97f26a2042d1fd99","name":"log","source":"97f26a2042d1fd99.txt","type":"text/plain","size":11541}],"parameters":[],"attachmentsCount":2,"shouldDisplayMessage":true,"stepsCount":0,"hasContent":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestOrder::0","time":{"start":1685024388837,"stop":1685024388837,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"labels":[{"name":"story","value":"Create an Order for new Customer "},{"name":"feature","value":"Order List"},{"name":"severity","value":"normal"},{"name":"suite","value":"TestOrder"},{"name":"subSuite","value":"TestOrder"},{"name":"host","value":"0day"},{"name":"thread","value":"4200-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"TestOrder"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"c843e2b48da4a994.json","parameterValues":[]}